add I3 support + update to formspecs v2

wsor4035 opened the pull request:
what the title says

fluxionary commented:
Fixing luacheck by making it assume the expected fields are present, is acceptable to me

fluxionary commented:
Should you also add i3 stylization to the filter injector formspecs?

wsor4035 commented:
> Should you also add i3 stylization to the filter injector formspecs?

🤦‍♂️ missed that, added https://github.com/mt-mods/pipeworks/pull/22/commits/7dfa021272909c42ee728aec9b2d25f54d4f3630

fluxionary commented:
> > Should you also add i3 stylization to the filter injector formspecs?
> 
> man_facepalming missed them, thanks for pointing them out, added [7dfa021](https://github.com/mt-mods/pipeworks/commit/7dfa021272909c42ee728aec9b2d25f54d4f3630)

![screenshot_20220416_112555](https://user-images.githubusercontent.com/25628292/163687063-a428bb63-8a21-452a-b40a-cdbee31fa071.png)

misplaced button in the stackwise injector

wsor4035 commented:
> > > Should you also add i3 stylization to the filter injector formspecs?
> > 
> > 
> > man_facepalming missed them, thanks for pointing them out, added [7dfa021](https://github.com/mt-mods/pipeworks/commit/7dfa021272909c42ee728aec9b2d25f54d4f3630)
> 
> ![screenshot_20220416_112555](https://user-images.githubusercontent.com/25628292/163687063-a428bb63-8a21-452a-b40a-cdbee31fa071.png)
> 
> misplaced button in the stackwise injector

what game and mods?

wsor4035 commented:
> > > Should you also add i3 stylization to the filter injector formspecs?
> > 
> > 
> > man_facepalming missed them, thanks for pointing them out, added [7dfa021](https://github.com/mt-mods/pipeworks/commit/7dfa021272909c42ee728aec9b2d25f54d4f3630)
> 
> ![screenshot_20220416_112555](https://user-images.githubusercontent.com/25628292/163687063-a428bb63-8a21-452a-b40a-cdbee31fa071.png)
> 
> misplaced button in the stackwise injector

fixed https://github.com/mt-mods/pipeworks/pull/22/commits/7426e11104f6aa639e36288fbc61780436200cda

fluxionary commented on teleport_tube.lua:
```diff
@@ -143,17 +143,19 @@ local function update_meta(meta, can_receive)
+		"label[7.0,0.5;"..S("Receive").."]"..
+		"image_button[7.0,0.75;1,0.6;pipeworks_button_" .. cr_state .. ".png;cr" .. (can_receive and 0 or 1) .. ";;;false;pipeworks_button_interm.png]"..
+		"button_exit[6.3,1.6;2,0.75;close;"..S("Close").."]"..
+		"label[0.5,2.7;"..itext.."]")
```
![screenshot_20220416_114501](https://user-images.githubusercontent.com/25628292/163687564-9257f188-987c-46a9-9d89-9ec64c696e03.png)

wsor4035 replied:
outdated

fluxionary commented:
how it looks on YL:

![screenshot_20220416_115043](https://user-images.githubusercontent.com/25628292/163687786-5d076b15-fb23-4820-a71e-2f94c1220a57.png)

how it looks now, after the last commit ([37eef73](https://github.com/mt-mods/pipeworks/commit/37eef736950f038e54fd74d11a5d905fb43abdad))

![screenshot_20220416_115059](https://user-images.githubusercontent.com/25628292/163687781-e408c04d-6fb3-4dc6-8c58-91fc401ab47b.png)
